### PR TITLE
Add a gitignore file and set up a linter for index.html page

### PR DESCRIPTION
**Pull request by alphayowakarindi**
I've added an index.html file with a header, set up a linter for HTML and CSS, and created a .gitignore file to ignore node_modules folder and test.md file from being pushed to github.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# .gitignore
+node_modules/
+test.md/
```
Comment (tufoinnkuo10) [inline .gitignore:3]: - [ ] please make sure your `test.md` file is actually being ignored. that is to say, it should not exist in your remote repository.

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @alphayowakarindi  

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Documented your work professionally
✅ Good commit messages
✅ Used correct README template
✅ Added gitignore file

**Required Changes** ♻️
Check the comments under the review
- [x] please ensure that your linters are correctly set up and passing. you can check how to do that [here](https://github.com/microverseinc/linters-config/tree/master/html-css).
- [x] Ensure that your Pull Request has a descriptive title telling you what changes are inside.

- [ ] please Add a description summary to your pull request. you can do that by clicking the 3dots and choosing edit. 👇🏾 ;


Screenshot (35)_LI

Optional suggestions

- [optional] please your Branch should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section').
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: .gitignore (hunk 1 above)
Comment (alphayowakarindi) [reply .gitignore:3]: @tufoinnkuo10 Thank you very much for the feedback. It's much appreciated. I've worked on each of the recommendations you recommended. I'd just like to add that the Linters/Webhint test fails wasn't due to the wrong setup but due to line breaks in my HTML document. After removing the line breaks all the tests passed.

Comment (alphayowakarindi) [reply .gitignore:3]: @tufoinnkuo10 My linter set up got a lot of mixups. I've redone it on my local machine and everything is working perfectly as expected. Should I delete this branch and make another pull request? Kindly advice?

Comment (tufoinnkuo10) [reply .gitignore:3]: - Thank you for implementing the changes. you are not supposed to create a new branch or pull a request before submitting again. please fix it and resubmit under this same pull request. 😄

Comment (alphayowakarindi) [reply .gitignore:3]: By that you mean I push the working code on the same branch from my local repository. Right?

File: index.html [hunk 2]
```diff
@@ -1,15 +1,18 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Hello Microverse</title>
     <link rel="stylesheet" href="style.css">
```
Comment (Sahar-AbdelSamad) [inline index.html:9]: - [ ] [OPTIONAL] As you can see in this line, you are referencing a CSS file with the name `style.css`, but your CSS file has the name 'styles.css' with `s`
This is not allowing the styling to be implemented. Kindly fix it so we can see the page with the yellow color 😄 as in the CSS file

File: .hintrc [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        "html-checker",
+        "meta-charset-utf-8",
+        "meta-viewport",
+        "no-inline-styles:error"
+    ]
+<<<<<<< HEAD
+}
+=======
+}
+>>>>>>> e118b06dfa104b0353219bb5aaf641a85818eec2
```
Comment (Sahar-AbdelSamad) [inline .hintrc:30]: - [ ] [OPTIONAL] I believe you forgot those lines. Please remove then 😃

**Review by Sahar-AbdelSamad — APPROVED** [review]
## STATUS APPROVED 🎉 🎉

Hi @alphayowakarindi 👋 👋 , 

Great work implementing the changes requested by the previous reviewer 👏
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

✔️  All linters passed 💯
✔️ Correct use of Github flow 💯

## Optional suggestions

- [x] [OPTIONAL] great job using microverse's README template and customizing it to your project 👏 👏 However you still have to customize your contact information 😃 kindly update it. This way when a recruiter likes your work he can contact you 😃 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (alphayowakarindi) [reply index.html:9]: Hi @Sahar-AbdelSamad, thanks for the feedback. The yellow color should be visible now :smiley:

File: .hintrc (hunk 3 above)
Comment (alphayowakarindi) [reply .hintrc]: @Sahar-AbdelSamad You are right. Fixed, thank you!

**Pull request merged**